gpt/public/tokenizer_config.txt
[{'<|endoftext|>': 0, '<|unk|>': 1, '!': 2, '"': 3, '$': 4, '%': 5, '&': 6, "'": 7, '(': 8, ')': 9, ',': 10, '-': 11, '.': 12, '0': 13, '1': 14, '2': 15, '3': 16, '4': 17, '5': 18, '6': 19, '7': 20, '8': 21, '9': 22, '?': 23, 'A': 24, 'B': 25, 'C': 26, 'D': 27, 'E': 28, 'F': 29, 'G': 30, 'H': 31, 'I': 32, 'J': 33, 'K': 34, 'L': 35, 'M': 36, 'N': 37, 'O': 38, 'P': 39, 'Q': 40, 'R': 41, 'S': 42, 'T': 43, 'U': 44, 'V': 45, 'W': 46, 'X': 47, 'Y': 48, 'Z': 49, '[': 50, ']': 51, '_': 52, 'a': 53, 'b': 54, 'c': 55, 'd': 56, 'e': 57, 'f': 58, 'g': 59, 'h': 60, 'i': 61, 'j': 62, 'k': 63, 'l': 64, 'm': 65, 'n': 66, 'o': 67, 'p': 68, 'q': 69, 'r': 70, 's': 71, 't': 72, 'u': 73, 'v': 74, 'w': 75, 'x': 76, 'y': 77, 'z': 78, '\xa0': 79, '£': 80, '̃': 81, '\u2002': 82, '\u2003': 83, '\u2004': 84, '\u2005': 85, '\u2006': 86, '\u2008': 87, '\u2009': 88, '\u200a': 89, '—': 90, '\u202f': 91, '€': 92, '\u3000': 93, 'G̃': 94, 'G̃t': 95, 'G̃a': 96, 'in': 97, 'on': 98, 'he': 99, 're': 100, 'G̃th': 101, 'G̃o': 102, 'G̃i': 103, 'G̃the': 104, 'G̃s': 105, 'G̃c': 106, 'G̃p': 107, 'G̃of': 108, 'G̃an': 109, 'G̃in': 110, 'ra': 111, 'G̃w': 112, 've': 113, 'de': 114, 'G̃f': 115, 'le': 116, 'G̃and': 117, 'G̃d': 118, 'G̃m': 119, 'G̃r': 120, 'G̃b': 121, 'G̃e': 122, 'G̃co': 123, 'la': 124, 'G̃re': 125, 'G̃to': 126, 'G̃pr': 127, 'ly': 128, 'G̃h': 129, 'us': 130, 'ge': 131, 'G̃l': 132, 'G̃T': 133, 'G̃fo': 134, 'G̃n': 135, 'G̃de': 136, 'ut': 137, 'G̃wi': 138, 'un': 139, 'G̃st': 140, 'G̃for': 141, 'G̃Th': 142, 'G̃pro': 143, 'G̃con': 144, 'G̃u': 145, 'G̃g': 146, 'G̃be': 147, 'G̃wit': 148, 'G̃with': 149, 'G̃is': 150, 'G̃di': 151, 'G̃pa': 152, 'va': 153, 'G̃ca': 154, 'G̃The': 155, 'G̃we': 156, 'G̃A': 157, 'G̃tha': 158, 'G̃mo': 159, 'G̃ma': 160, 'G̃on': 161, 'G̃v': 162, 'G̃me': 163, 'G̃su': 164, 'G̃C': 165, 'G̃G': 166, 'G̃as': 167, 'G̃se': 168, 'G̃I': 169, 'G̃ar': 170, 'G̃ex': 171, 'G̃S': 172, 'G̃wa': 173, 'G̃that': 174, 'G̃com': 175, 'G̃res': 176, 'G̃ha': 177, 'G̃po': 178, 'G̃tr': 179, 'G̃wh': 180, 'G̃si': 181, 'G̃P': 182, 'G̃was': 183, 'G̃by': 184, 'G̃fr': 185, 'G̃no': 186, 'G̃al': 187, 'G̃1': 188, 'G̃int': 189, 'G̃ac': 190, 'G̃M': 191, 'G̃are': 192, 'G̃pre': 193, 'G̃ch': 194, 'G̃us': 195, 'G̃G̃': 574, 'G̃im': 197, 'G̃wer': 198, 'G̃were': 199, 'G̃pe': 200, 'G̃comp': 201, 'G̃te': 202, 'G̃ne': 203, 'G̃or': 204, 'G̃dis': 205, 'ke': 206, 'G̃li': 207, 'G̃le': 208, 'G̃at': 209, 'G̃ra': 210, 'G̃fi': 211, 'G̃en': 212, 'G̃thi': 213, 'G̃2': 214, 'G̃fro': 215, 'G̃so': 216, 'G̃D': 217, 'G̃from': 218, 'G̃In': 219, 'G̃pat': 220, 'G̃R': 221, 'G̃H': 222, 'G̃per': 223, 'G̃cont': 224, 'G̃sh': 225, 'G̃gr': 226, 'G̃stu': 227, 'G̃E': 228, 'G̃stud': 229, 'G̃sp': 230, 'G̃lo': 231, 'G̃F': 232, 'G̃inte': 233, 'G̃B': 234, 'G̃un': 235, 'G̃exp': 236, 'G̃ad': 237, 'G̃inc': 238, 'G̃this': 239, 'G̃W': 240, 'G̃mi': 241, 'G̃sy': 242, 'G̃hi': 243, 'G̃fa': 244, 'G̃tra': 245, 'G̃imp': 246, 'G̃la': 247, 'G̃ap': 248, 'G̃he': 249, 'ze': 250, 'G̃va': 251, 'G̃use': 252, 'G̃ce': 253, 'G̃whi': 254, 'G̃app': 255, 'G̃ba': 256, 'G̃cha': 257, 'G̃mod': 258, 'G̃str': 259, 'G̃N': 260, 'G̃cl': 261, 'G̃sta': 262, 'G̃ef': 263, 'G̃eff': 264, 'G̃da': 265, 'G̃it': 266, 'G̃ge': 267, 'G̃par': 268, 'G̃O': 269, 'G̃met': 270, 'G̃pati': 271, 'G̃patie': 272, 'G̃patien': 273, 'G̃patient': 274, 'G̃sho': 275, 'G̃can': 276, 'G̃rec': 277, 'G̃inter': 278, 'G̃L': 279, 'G̃cons': 280, 'G̃ind': 281, 'G̃fu': 282, 'G̃bo': 283, 'G̃inf': 284, 'G̃whic': 285, 'G̃which': 286, 'G̃sc': 287, 'G̃dif': 288, 'G̃diff': 289, '00': 290, 'G̃pres': 291, 'G̃rel': 292, 'G̃mode': 293, 'G̃ana': 294, 'G̃hig': 295, 'G̃high': 296, 'G̃vi': 297, 'G̃wo': 298, 'G̃patients': 299, 'G̃ev': 300, 'G̃do': 301, 'G̃hav': 302, 'G̃mu': 303, 'G̃ph': 304, 'G̃ho': 305, 'G̃not': 306, 'G̃gro': 307, 'G̃ass': 308, 'G̃resu': 309, 'G̃sys': 310, 'G̃syst': 311, 'G̃gen': 312, 'G̃resul': 313, 'G̃result': 314, 'G̃ob': 315, 'G̃syste': 316, 'G̃system': 317, 'G̃anal': 318, 'G̃bet': 319, 'G̃contr': 320, 'G̃study': 321, 'G̃sig': 322, 'G̃have': 323, 'G̃Co': 324, 'G̃spe': 325, 'G̃pl': 326, 'G̃sign': 327, 'G̃dev': 328, 'G̃sa': 329, 'G̃ab': 330, 'G̃k': 331, 'G̃ti': 332, 'G̃3': 333, 'G̃ou': 334, 'G̃analy': 335, 'G̃bu': 336, 'G̃effe': 337, 'G̃effec': 338, 'G̃effect': 339, 'G̃diffe': 340, 'G̃differ': 341, 'G̃cel': 342, 'G̃op': 343, 'G̃tran': 344, 'G̃trans': 345, 'G̃cell': 346, 'G̃act': 347, 'G̃des': 348, 'G̃pla': 349, 'G̃differe': 350, 'G̃has': 351, 'G̃mea': 352, 'G̃model': 353, 'G̃differen': 354, 'G̃meth': 355, 'G̃pos': 356, 'G̃prop': 357, 'G̃dat': 358, 'G̃cr': 359, 'G̃rep': 360, 'G̃prese': 361, 'G̃show': 362, 'G̃rat': 363, 'G̃det': 364, 'G̃ag': 365, 'za': 366, 'G̃el': 367, 'G̃fe': 368, 'G̃thr': 369, 'G̃gene': 370, 'G̃mor': 371, 'G̃du': 372, 'G̃ve': 373, 'je': 374, 'G̃Thi': 375, 'G̃bi': 376, 'G̃q': 377, 'G̃presen': 378, 'G̃data': 379, 'G̃car': 380, 'G̃spec': 381, 'G̃metho': 382, 'G̃qu': 383, 'G̃betw': 384, 'G̃betwe': 385, 'G̃betwee': 386, 'G̃between': 387, 'G̃sub': 388, 'G̃method': 389, 'G̃incr': 390, 'G̃Re': 391, 'G̃incre': 392, 'ing': 393, 'G̃U': 394, 'G̃reg': 395, 'G̃We': 396, 'G̃ro': 397, 'G̃increa': 398, 'G̃increas': 399, 'G̃all': 400, 'G̃bas': 401, 'G̃This': 402, 'G̃acti': 403, 'G̃usi': 404, 'G̃usin': 405, 'G̃using': 406, 'G̃inv': 407, 'G̃resp': 408, 'G̃tre': 409, 'G̃acc': 410, 'G̃sur': 411, 'G̃deve': 412, 'G̃devel': 413, 'G̃develo': 414, 'G̃develop': 415, 'G̃cu': 416, 'G̃y': 417, 'G̃af': 418, 'G̃whe': 419, 'G̃part': 420, 'G̃rea': 421, 'G̃conc': 422, 'G̃dete': 423, 'G̃und': 424, 'G̃rela': 425, 'G̃ther': 426, 'G̃bee': 427, 'G̃thei': 428, 'G̃their': 429, 'G̃compa': 430, 'G̃0': 431, 'G̃results': 432, 'G̃grou': 433, 'G̃unde': 434, 'G̃been': 435, 'G̃man': 436, 'G̃cor': 437, 'G̃ta': 438, 'G̃signi': 439, 'G̃expe': 440, 'G̃under': 441, 'G̃signif': 442, 'G̃signifi': 443, 'G̃signific': 444, 'G̃significa': 445, 'G̃significan': 446, 'G̃nu': 447, 'G̃fac': 448, 'G̃var': 449, 'G̃proc': 450, 'G̃20': 451, 'G̃thes': 452, 'ion': 453, 'G̃used': 454, 'G̃analys': 455, 'G̃pu': 456, 'G̃tw': 457, 'G̃present': 458, 'G̃ant': 459, 'G̃proce': 460, 'G̃relat': 461, 'G̃cou': 462, 'G̃es': 463, 'G̃wor': 464, 'G̃than': 465, 'G̃these': 466, 'G̃perf': 467, 'G̃group': 468, 'G̃hea': 469, 'G̃significant': 470, 'G̃trea': 471, 'G̃4': 472, 'G̃activ': 473, 'G̃treat': 474, 'G̃tim': 475, 'G̃em': 476, 'G̃vari': 477, 'G̃out': 478, 'G̃ri': 479, 'G̃contro': 480, 'G̃am': 481, 'G̃na': 482, 'G̃als': 483, 'G̃also': 484, 'G̃increase': 485, 'G̃appr': 486, 'G̃sim': 487, 'G̃prot': 488, 'G̃suc': 489, 'G̃compar': 490, 'G̃hy': 491, 'G̃ea': 492, 'G̃time': 493, 'G̃ov': 494, 'G̃fl': 495, 'G̃perfo': 496, 'G̃prov': 497, 'G̃perfor': 498, 'G̃control': 499, 'G̃br': 500, 'G̃ele': 501, 'G̃perform': 502, 'G̃more': 503, 'G̃lea': 504, 'G̃different': 505, 'G̃appro': 506, 'G̃two': 507, 'G̃fun': 508, 'G̃comm': 509, 'G̃5': 510, 'G̃dec': 511, 'G̃sup': 512, 'G̃compl': 513, 'G̃G̃T': 571, 'G̃indi': 515, 'G̃speci': 516, 'G̃chan': 517, 'G̃pol': 518, 'G̃exper': 519, 'G̃ove': 520, 'G̃base': 521, 'G̃over': 522, 'G̃col': 523, 'G̃stat': 524, 'G̃ext': 525, 'G̃but': 526, 'G̃dr': 527, 'G̃low': 528, 'G̃meas': 529, 'G̃cas': 530, 'G̃cond': 531, 'G̃measu': 532, 'G̃measur': 533, 'G̃pap': 534, 'G̃ser': 535, 'G̃lev': 536, 'G̃id': 537, 'G̃ins': 538, 'G̃ot': 539, 'G̃appl': 540, 'G̃tec': 541, 'G̃reco': 542, 'G̃form': 543, 'G̃disc': 544, 'G̃leve': 545, 'G̃analysi': 546, 'G̃tech': 547, 'G̃its': 548, 'G̃techn': 549, 'G̃V': 550, 'G̃fou': 551, 'G̃propo': 552, 'G̃provi': 553, 'G̃oth': 554, 'G̃red': 555, 'G̃othe': 556, 'G̃other': 557, 'G̃analysis': 558, 'G̃dist': 559, 'G̃G̃Th': 572, 'G̃pape': 561, 'G̃paper': 562, 'G̃parti': 563, 'G̃experi': 564, 'G̃ide': 565, 'G̃pri': 566, 'G̃conce': 567, 'G̃rev': 568, 'G̃level': 569, 'G̃one': 573, 'G̃proces': 575, 'G̃process': 576, 'G̃fin': 577, 'G̃ga': 578, 'G̃val': 579, 'G̃rese': 580, 'G̃prob': 581, 'G̃mic': 582, 'G̃measure': 583, 'G̃tes': 584, 'G̃test': 585, 'G̃rece': 586, 'G̃qua': 587, 'G̃au': 588, 'G̃bot': 589, 'G̃incl': 590, 'G̃consi': 591, 'G̃prod': 592, 'G̃inclu': 593, 'G̃studi': 594, 'G̃produ': 595, 'G̃produc': 596, 'G̃studie': 597, 'G̃provid': 598, 'G̃case': 599, 'G̃Pr': 600, 'G̃anti': 601, 'G̃mat': 602, 'G̃func': 603, 'G̃funct': 604, 'G̃functi': 605, 'G̃functio': 606, 'G̃function': 607, 'G̃aft': 608, 'G̃afte': 609, 'G̃after': 610, 'G̃new': 611, 'G̃St': 612, 'G̃ye': 613, 'G̃dur': 614, 'G̃cells': 615, 'G̃treatm': 616, 'G̃treatme': 617, 'G̃treatmen': 618, 'G̃treatment': 619, 'G̃appli': 620, 'G̃mul': 621, 'G̃sam': 622, 'G̃dia': 623, 'G̃gener': 624, 'G̃both': 625, 'G̃prote': 626, 'G̃tu': 627, 'G̃non': 628, 'G̃redu': 629, 'G̃mult': 630, 'G̃multi': 631, 'G̃bl': 632, 'G̃char': 633, 'G̃fact': 634, 'G̃inve': 635, 'G̃med': 636, 'G̃sol': 637, 'G̃relati': 638, 'G̃genera': 639, 'G̃exa': 640, 'G̃propos': 641, 'G̃reduc': 642, 'G̃sec': 643, 'G̃includ': 644, 'G̃medi': 645, 'G̃add': 646, 'G̃work': 647, 'G̃Me': 648, 'G̃may': 649, 'G̃6': 650, 'G̃such': 651, 'G̃art': 652, 'G̃obs': 653, 'G̃respo': 654, 'G̃respon': 655, 'G̃chang': 656, 'G̃Ma': 657, 'G̃cli': 658, 'G̃desi': 659, 'G̃based': 660, 'G̃G̃G': 661, 'G̃propose': 662, 'G̃age': 663, 'G̃dep': 664, 'G̃provide': 665, 'G̃impr': 666, 'G̃mon': 667, 'G̃stru': 668, 'G̃wel': 669, 'G̃into': 670, 'G̃supp': 671, 'G̃comple': 672, 'G̃Ca': 673, 'G̃est': 674, 'G̃dem': 675, 'G̃partic': 676, 'G̃iden': 677, 'G̃ident': 678, 'G̃identi': 679, 'G̃att': 680, 'G̃struc': 681, 'G̃struct': 682, 'G̃structu': 683, 'G̃structur': 684, 'G̃elec': 685, 'G̃elect': 686, 'G̃exam': 687, 'G̃asso': 688, 'G̃Ho': 689, 'G̃resea': 690, 'G̃resear': 691, 'G̃researc': 692, 'G̃research': 693, 'G̃assoc': 694, 'G̃associ': 695, 'G̃associa': 696, 'G̃associat': 697, 'G̃yea': 698, 'G̃change': 699, 'G̃num': 700, 'G̃impro': 701, 'G̃para': 702, 'G̃lin': 703, 'G̃G̃G̃': 4392, 'G̃improv': 705, 'G̃thro': 706, 'G̃Mo': 707, 'G̃gra': 708, 'G̃duri': 709, 'G̃impo': 710, 'G̃durin': 711, 'G̃during': 712, 'G̃corr': 713, 'G̃ver': 714, 'G̃well': 715, 'G̃De': 716, 'G̃rate': 717, 'G̃opt': 718, 'G̃facto': 719, 'G̃inves': 720, 'G̃invest': 721, 'G̃year': 722, 'G̃compare': 723, 'G̃factor': 724, 'G̃gl': 725, 'G̃hu': 726, 'G̃10': 727, 'G̃sen': 728, 'G̃thre': 729, 'G̃desig': 730, 'G̃design': 731, 'G̃who': 732, 'G̃onl': 733, 'G̃approa': 734, 'G̃approac': 735, 'G̃approach': 736, 'G̃electr': 737, 'G̃opti': 738, 'G̃mos': 739, 'G̃19': 740, 'G̃eva': 741, 'G̃dise': 742, 'G̃micr': 743, 'G̃micro': 744, 'G̃prog': 745, 'G̃obse': 746, 'G̃go': 747, 'G̃obser': 748, 'G̃observ': 749, 'G̃An': 750, 'G̃cur': 751, 'G̃had': 752, 'G̃aut': 753, 'G̃It': 754, 'G̃prev': 755, 'G̃sel': 756, 'G̃disea': 757, 'G̃diseas': 758, 'G̃disease': 759, 'G̃up': 760, 'G̃Con': 761, 'G̃def': 762, 'G̃deter': 763, 'G̃ran': 764, 'G̃fre': 765, 'G̃activi': 766, 'G̃ty': 767, 'G̃fir': 768, 'G̃chara': 769, 'G̃charac': 770, 'G̃charact': 771, 'G̃impor': 772, 'G̃characte': 773, 'G̃character': 774, 'G̃cla': 775, 'G̃import': 776, 'G̃compo': 777, 'G̃corre': 778, 'G̃most': 779, 'G̃Al': 780, 'G̃respons': 781, 'G̃J': 782, 'G̃activit': 783, 'G̃Se': 784, 'G̃eval': 785, 'G̃evalu': 786, 'G̃evalua': 787, 'G̃abo': 788, 'G̃evaluat': 789, 'G̃importa': 790, 'G̃importan': 791, 'G̃cal': 792, 'G̃developm': 793, 'G̃developme': 794, 'G̃developmen': 795, 'G̃development': 796, 'G̃G̃The': 797, 'G̃repo': 798, 'G̃Pa': 799, 'G̃clin': 800, 'G̃foun': 801, 'G̃cy': 802, 'G̃fol': 803, 'G̃found': 804, 'G̃determ': 805, 'G̃determi': 806, 'G̃determin': 807, 'G̃improve': 808, 'G̃clini': 809, 'G̃clinic': 810, 'G̃repor': 811, 'G̃report': 812, 'G̃investi': 813, 'G̃Ch': 814, 'G̃ope': 815, 'G̃probl': 816, 'G̃typ': 817, 'G̃loc': 818, 'G̃proble': 819, 'G̃problem': 820, 'G̃investig': 821, 'G̃investiga': 822, 'G̃investigat': 823, 'G̃av': 824, 'G̃identif': 825, 'G̃asse': 826, 'G̃wil': 827, 'G̃his': 828, 'G̃throu': 829, 'G̃throug': 830, 'G̃sens': 831, 'G̃through': 832, 'G̃tho': 833, 'G̃kn': 834, 'G̃j': 835, 'G̃characteri': 836, 'G̃ref': 837, 'G̃there': 838, 'G̃sug': 839, 'G̃heal': 840, 'G̃To': 841, 'G̃only': 842, 'G̃stra': 843, 'G̃hyp': 844, 'G̃regi': 845, 'G̃lar': 846, 'G̃G̃A': 847, 'G̃chi': 848, 'G̃impl': 849, 'G̃amo': 850, 'G̃sm': 851, 'G̃ris': 852, 'G̃valu': 853, 'G̃pot': 854, 'G̃const': 855, 'G̃intr': 856, 'G̃how': 857, 'G̃bio': 858, 'G̃ima': 859, 'G̃min': 860, 'G̃info': 861, 'G̃sin': 862, 'G̃expr': 863, 'G̃expre': 864, 'G̃expres': 865, 'G̃express': 866, 'G̃peri': 867, 'G̃imag': 868, 'G̃9': 869, 'G̃infor': 870, 'G̃inform': 871, 'G̃hum': 872, 'G̃performa': 873, 'G̃Fo': 874, 'G̃compared': 875, 'G̃He': 876, 'G̃performan': 877, 'G̃performanc': 878, 'G̃performance': 879, 'G̃foll': 880, 'G̃detec': 881, 'G̃detect': 882, 'G̃structure': 883, 'G̃experim': 884, 'G̃experime': 885, 'G̃experimen': 886, 'G̃experiment': 887, 'G̃coll': 888, 'G̃sugg': 889, 'G̃sugge': 890, 'G̃sugges': 891, 'G̃suggest': 892, 'G̃som': 893, 'G̃varia': 894, 'G̃eve': 895, 'G̃studies': 896, 'G̃techni': 897, 'G̃req': 898, 'G̃requ': 899, 'G̃conf': 900, 'G̃healt': 901, 'G̃health': 902, 'G̃highe': 903, 'G̃desc': 904, 'G̃indic': 905, 'G̃numb': 906, 'G̃syn': 907, 'G̃imm': 908, 'G̃ai': 909, 'G̃protei': 910, 'G̃protein': 911, 'G̃sev': 912, 'G̃numbe': 913, 'G̃number': 914, 'G̃cap': 915, 'G̃demo': 916, 'G̃pred': 917, 'G̃seve': 918, 'G̃when': 919, 'G̃mar': 920, 'G̃indu': 921, 'G̃fra': 922, 'G̃subs': 923, 'G̃kno': 924, 'G̃larg': 925, 'G̃large': 926, 'G̃ci': 927, 'G̃7': 928, 'G̃condi': 929, 'G̃condit': 930, 'G̃conditi': 931, 'G̃conditio': 932, 'G̃condition': 933, 'G̃follo': 934, 'G̃follow': 935, 'G̃expl': 936, 'G̃firs': 937, 'G̃first': 938, 'G̃uni': 939, 'G̃loca': 940, 'G̃applic': 941, 'G̃applica': 942, 'G̃oc': 943, 'G̃rem': 944, 'G̃exc': 945, 'G̃consid': 946, 'G̃conside': 947, 'G̃consider': 948, 'G̃value': 949, 'G̃end': 950, 'G̃mai': 951, 'G̃pote': 952, 'G̃poten': 953, 'G̃descr': 954, 'G̃some': 955, 'G̃descri': 956, 'G̃infl': 957, 'G̃meta': 958, 'G̃progr': 959, 'G̃8': 960, 'G̃will': 961, 'G̃huma': 962, 'G̃human': 963, 'G̃ste': 964, 'G̃clinica': 965, 'G̃clinical': 966, 'G̃neu': 967, 'G̃informa': 968, 'G̃vo': 969, 'G̃years': 970, 'G̃net': 971, 'G̃proposed': 972, 'G̃requi': 973, 'G̃know': 974, 'G̃response': 975, 'G̃potent': 976, 'G̃our': 977, 'G̃requir': 978, 'G̃effects': 979, 'G̃product': 980, 'G̃associate': 981, 'G̃indica': 982, 'G̃three': 983, 'G̃indicat': 984, 'G̃Res': 985, 'G̃les': 986, 'G̃main': 987, 'G̃Di': 988, 'G̃sch': 989, 'G̃increased': 990, 'G̃informat': 991, 'G̃informati': 992, 'G̃blo': 993, 'G̃associated': 994, 'G̃How': 995, 'G̃exami': 996, 'G̃examin': 997, 'G̃posi': 998, 'G̃samp': 999, 'G̃sampl': 1000, 'G̃sca': 1001, 'G̃electro': 1002, 'G̃nor': 1003, 'G̃posit': 1004, 'G̃asses': 1005, 'G̃dy': 1006, 'G̃assess': 1007, 'G̃informatio': 1008, 'G̃information': 1009, 'G̃stre': 1010, 'G̃inh': 1011, 'G̃mec': 1012, 'G̃risk': 1013, 'G̃abou': 1014, 'G̃effi': 1015, 'G̃bec': 1016, 'G̃effic': 1017, 'G̃find': 1018, 'G̃about': 1019, 'G̃mech': 1020, 'G̃mecha': 1021, 'G̃conte': 1022, 'G̃soc': 1023, 'G̃techniq': 1024, 'G̃techniqu': 1025, 'G̃technique': 1026, 'G̃clas': 1027, 'G̃mechan': 1028, 'G̃pow': 1029, 'G̃Si': 1030, 'G̃mean': 1031, 'G̃potenti': 1032, 'G̃applicat': 1033, 'G̃mechani': 1034, 'G̃class': 1035, 'G̃require': 1036, 'G̃alg': 1037, 'G̃systems': 1038, 'G̃applicati': 1039, 'G̃important': 1040, 'G̃potentia': 1041, 'G̃applicatio': 1042, 'G̃application': 1043, 'G̃grow': 1044, 'G̃positi': 1045, 'G̃rad': 1046, 'G̃type': 1047, 'G̃pho': 1048, 'G̃soci': 1049, 'G̃significantl': 1050, 'G̃significantly': 1051, 'G̃activity': 1052, 'G̃netw': 1053, 'G̃netwo': 1054, 'G̃networ': 1055, 'G̃network': 1056, 'G̃demon': 1057, 'G̃occ': 1058, 'G̃ter': 1059, 'G̃nee': 1060, 'G̃addi': 1061, 'G̃pha': 1062, 'G̃demons': 1063, 'G̃demonst': 1064, 'G̃need': 1065, 'G̃Fi': 1066, 'G̃demonstr': 1067, 'G̃demonstra': 1068, 'G̃levels': 1069, 'G̃sever': 1070, 'G̃curr': 1071, 'G̃demonstrat': 1072, 'G̃simu': 1073, 'G̃showe': 1074, 'G̃simul': 1075, 'G̃spa': 1076, 'G̃adv': 1077, 'G̃showed': 1078, 'G̃state': 1079, 'G̃relatio': 1080, 'G̃relation': 1081, 'G̃line': 1082, 'G̃Howe': 1083, 'G̃Howev': 1084, 'G̃Howeve': 1085, 'G̃However': 1086, 'G̃potential': 1087, 'G̃they': 1088, 'G̃arti': 1089, 'G̃obt': 1090, 'G̃Thes': 1091, 'G̃post': 1092, 'G̃specif': 1093, 'G̃As': 1094, 'G̃So': 1095, 'G̃obta': 1096, 'G̃obtai': 1097, 'G̃obtain': 1098, 'G̃These': 1099, 'G̃nat': 1100, 'G̃cri': 1101, 'G̃exte': 1102, 'G̃higher': 1103, 'G̃conv': 1104, 'G̃predi': 1105, 'G̃area': 1106, 'G̃beh': 1107, 'G̃rol': 1108, 'G̃tri': 1109, 'G̃powe': 1110, 'G̃specifi': 1111, 'G̃power': 1112, 'G̃reso': 1113, 'G̃cau': 1114, 'G̃Ba': 1115, 'G̃observe': 1116, 'G̃resi': 1117, 'G̃reduce': 1118, 'G̃sti': 1119, 'G̃phy': 1120, 'G̃pra': 1121, 'G̃tem': 1122, 'G̃extr': 1123, 'G̃temp': 1124, 'G̃fea': 1125, 'G̃radi': 1126, 'G̃ret': 1127, 'G̃Ne': 1128, 'G̃gi': 1129, 'G̃algo': 1130, 'G̃comb': 1131, 'G̃algor': 1132, 'G̃algori': 1133, 'G̃algorit': 1134, 'G̃eac': 1135, 'G̃algorith': 1136, 'G̃each': 1137, 'G̃Un': 1138, 'G̃algorithm': 1139, 'G̃set': 1140, 'G̃chil': 1141, 'G̃determine': 1142, 'G̃ed': 1143, 'G̃et': 1144, 'G̃wher': 1145, 'G̃child': 1146, 'G̃differenc': 1147, 'G̃where': 1148, 'G̃difference': 1149, 'G̃distr': 1150, 'G̃predic': 1151, 'G̃reac': 1152, 'G̃suppo': 1153, 'G̃commu': 1154, 'G̃Pro': 1155, 'G̃No': 1156, 'G̃curre': 1157, 'G̃curren': 1158, 'G̃path': 1159, 'G̃withi': 1160, 'G̃emp': 1161, 'G̃pi': 1162, 'G̃plan': 1163, 'G̃within': 1164, 'G̃addit': 1165, 'G̃acco': 1166, 'G̃additi': 1167, 'G̃effecti': 1168, 'G̃sample': 1169, 'G̃thera': 1170, 'G̃effectiv': 1171, 'G̃therap': 1172, 'G̃commun': 1173, 'G̃expressi': 1174, 'G̃diag': 1175, 'G̃effective': 1176, 'G̃predict': 1177, 'G̃role': 1178, 'G̃oper': 1179, 'G̃respe': 1180, 'G̃respec': 1181, 'G̃respect': 1182, 'G̃Su': 1183, 'G̃opera': 1184, 'G̃eq': 1185, 'G̃concen': 1186, 'G̃investigate': 1187, 'G̃communi': 1188, 'G̃concent': 1189, 'G̃infe': 1190, 'G̃aff': 1191, 'G̃concentr': 1192, 'G̃specific': 1193, 'G̃equ': 1194, 'G̃beha': 1195, 'G̃current': 1196, 'G̃suppor': 1197, 'G̃inst': 1198, 'G̃support': 1199, 'G̃caus': 1200, 'G̃concentra': 1201, 'G̃concentrat': 1202, 'G̃prac': 1203, 'G̃env': 1204, 'G̃pract': 1205, 'G̃practi': 1206, 'G̃demonstrate': 1207, 'G̃poly': 1208, 'G̃neur': 1209, 'G̃repr': 1210, 'G̃behav': 1211, 'G̃conve': 1212, 'G̃even': 1213, 'G̃operat': 1214, 'ive': 1215, 'G̃prope': 1216, 'G̃allo': 1217, 'G̃impa': 1218, 'G̃poss': 1219, 'G̃qual': 1220, 'G̃cat': 1221, 'G̃quali': 1222, 'G̃describ': 1223, 'G̃serv': 1224, 'G̃canc': 1225, 'G̃include': 1226, 'G̃cance': 1227, 'G̃ec': 1228, 'G̃revi': 1229, 'G̃sit': 1230, 'G̃freq': 1231, 'G̃frequ': 1232, 'G̃freque': 1233, 'G̃frequen': 1234, 'G̃lon': 1235, 'G̃regio': 1236, 'G̃behavi': 1237, 'G̃behavio': 1238, 'G̃discu': 1239, 'G̃reve': 1240, 'G̃see': 1241, 'G̃sma': 1242, 'G̃param': 1243, 'G̃care': 1244, 'G̃Ther': 1245, 'G̃region': 1246, 'G̃mana': 1247, 'G̃combi': 1248, 'G̃lit': 1249, 'G̃manag': 1250, 'G̃theo': 1251, 'G̃combin': 1252, 'G̃diagn': 1253, 'G̃diagno': 1254, 'G̃diagnos': 1255, 'G̃Com': 1256, 'G̃conta': 1257, 'G̃cen': 1258, 'G̃dir': 1259, 'G̃discus': 1260, 'G̃che': 1261, 'G̃discuss': 1262, 'G̃hist': 1263, 'G̃long': 1264, 'G̃cancer': 1265, 'G̃methods': 1266, 'G̃surf': 1267, 'G̃observed': 1268, 'G̃additio': 1269, 'G̃addition': 1270, 'G̃optim': 1271, 'G̃envi': 1272, 'G̃surfa': 1273, 'G̃surfac': 12

gpt/config.py
import sys
import argparse
import torch

parser = argparse.ArgumentParser()
parser.add_argument("-l", "--local-pretrain", action="store_true")
local = parser.parse_args(sys.argv[1:]).local_pretrain

if local:
    settings = {
        "n_vocab": 45098,
        "n_ctx": 256,
        "n_embd": 1024,
        "n_head": 16,
        "n_layer": 24,
    }
else:
    settings = {
        "n_vocab": 50257,
        "n_ctx": 1024,
        "n_embd": 1024,
        "n_head": 16,
        "n_layer": 24,
    }

# MODEL CONFIG

VOCAB_SIZE = settings["n_vocab"]
CONTEXT_LEN = settings["n_ctx"]
EMB_DIM = settings["n_embd"]
N_HEADS = settings["n_head"]
N_LAYERS = settings["n_layer"]
DROPOUT = 0.1
QKV_BIAS = False if VOCAB_SIZE == 45098 else True
TRAIN_RATIO = 0.9
BATCH_SIZE = 8
OPENAI_MODEL_SIZE = "355M"

# DEVICE AND DOWNLOAD CONFIG

DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
CORES = 10
SAMPLE_BYTES = 10 * 1024 * 1024
TOKENIZER_BYTES = 30 * 1024 * 1024
PRETRAINING_MB = 512 * 1024 * 1024

# DATA FILES

SAMPLE_FILE = "data/academic_sample.txt"
TOKENIZER_FILE = "data/academic_tokenizing.txt"
PRETRAINING_FILE = "data/academic_pretraining.txt"
SPAM_FILE = "data/spam-data.txt"

# PUBLIC FILES

TOKENIZER_CONFIG = "public/tokenizer_config.txt"
SPAM_TRAIN_CSV = "public/spam-train.csv"
SPAM_VAL_CSV = "public/spam-val.csv"
SPAM_TEST_CSV = "public/spam-test.csv"
INSTRUCTION_FILE = "public/instruction-data.json"

# MODEL FILES

MODEL_FILE = "models/model.pth"
OPENAI_WEIGHTS_MODEL_FILE = "models/openai_weights_model.pth"
CLASSIFICATION_MODEL_FILE = "models/classifier_model.pth"
INSTRUCTION_MODEL_FILE = "models/instruction_model.pth"

# OTHER

DATA = [PRETRAINING_FILE, PRETRAINING_MB]
